spec/mailers/subscription_mailer_spec.rb

```diff
@@ -41,11 +41,9 @@
     end
 
     describe "linking to order page" do
-      let(:order_link_href) { "href=\"#{order_url(order)}\"" }
-      let(:order_link_style) { "style='[^']+'" }
-
       let(:shop) { create(:enterprise, allow_order_changes: true) }
 
+      let(:content) { Capybara::Node::Simple.new(body) }
```

Comment:
Clever! Now we can use capybara expectations just like on a web page. ✨ 

🤔 I wonder if we should build this in to a mailer spec helper...

Reply:
Do you know what the style check was for? I doubt it's relevant to these test expectations so I'm happy to see it go.

Reply:
I think that it was just part of the regular expression to match the link but nothing else. It wasn't checking the style.